Neos.Neos/Classes/Fusion/ConvertUrisImplementation.php

```diff
@@ -142,7 +142,7 @@ protected function replaceLinkTargets($processedContent)
         $controllerContext = $this->runtime->getControllerContext();
         $host = $controllerContext->getRequest()->getHttpRequest()->getUri()->getHost();
         $processedContent = preg_replace_callback(
-            '~<a.*?href="(.*?)".*?>~i',
+            '~<a(?: [^>]*)? href="(.*?)".*?>~i',
```

Comment:
Good catch, but I'd change the whitespaces in both regexes to `\s+` so they also match tab, line breaks and/or multiple whitespaces.